octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,7 +1,157 @@
 $(function() {
     function V8ThemeViewModel(parameters) {
         var self = this;
-        self.temp = parameters[0]
+        self.temperature = parameters[0];
+        self.terminal = parameters[1];
+        
+        var Colors = ["#fbc08d", "#fde0c8", "#ffb6a6", "#ffdbd4"];
+        self.terminal.lineCount = ko.computed(function() {
+            var total = self.terminal.log().length;
+            var displayed = _.filter(self.terminal.displayedLines(), function(entry) {
+                return entry.type == "line"
```

Comment:
You should be using triple equals, `===` and `!==`, for all equality checks.  The only exception is something like `if (foo == null)`, which checks against either `null` or `undefined`.